### PR TITLE
Milestone 1: Finished creating mobile-menu

### PR DESCRIPTION
**Pull request by prg-04**
**### In this Milestone, I have done the following step to create the Mobile-menu:**

- [ ]  Use javascript to hide and display hamburger menu

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,17 +12,22 @@
...
+     <header class="top-nav">
+      <div class="logo">
+        <h1>Evans Karanja</h1>
+      </div>
+      <nav class="nav-links">
+        <a href="#">Portfolio</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+        <a href="#"><i class="fa-solid fa-envelope"></i></a>
+      </nav>
```
Comment (franclobo) [inline index.html:24]: - [ ] Please 🙏🏽 make sure when the user clicks (or taps) any of the mobile menu options, the mobile menu disappears, it is a requirement.

- [ ] Kindly correct the font weight and erase the last icon in the mobile version, it is a requirement that you must stick to the design as much as possible (e.g., font, colors, images, text, margins) using [the templates in Figma](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 👇🏽 

Figma  design

image

Yours

image

File: index.html [hunk 2]
```diff
@@ -12,17 +12,22 @@
...
-      </ul>
+     <header class="top-nav">
+      <div class="logo">
+        <h1>Evans Karanja</h1>
+      </div>
+      <nav class="nav-links">
+        <a href="#">Portfolio</a>
+        <a href="#">About</a>
+        <a href="#">Contact</a>
+        <a href="#"><i class="fa-solid fa-envelope"></i></a>
```
Comment (franclobo) [inline index.html:23]: - [ ] When the user clicks (or taps) any of the mobile menu options, a correct part of the page is displayed. Please 🙏🏽 complete the `href` to go to the corresponding section, we could use `href="#portfolio"`. First, make sure that the corresponding section has an `id`. We could use `<h2 id="portfolio">My Recent Works<h2>` and so on.

**Review by franclobo — CHANGES_REQUESTED** [review]
Hi @prg-04, 

# Required Changes ♻️
Good job so far!
You are doing it well 👍🏽, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors ✔️ 
- Correct GitHub flow ✔️ 
- Descriptive PR ✔️ 

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

You can also consider:
- [ ] Please 🙏🏽 confirm that all the links in your README file work as expected.
- [ ] The LICENSE file should go to the root folder, kindly change the name MIT.md to LICENSE put it in the root folder and erase the .LICENSE folder. It going to fix the behavior of the `[MIT](./LICENSE)`.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Munsa1 — APPROVED** [review]
Hi @prg-04, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
:heavy_check_mark: Descriptive Readme
:heavy_check_mark: Passing Linters
:heavy_check_mark: Correct Gitflow

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**